xml/entity-decl.ent

```diff
@@ -189,6 +189,7 @@ suggested official replacement from rsalevsky. -->
 <!ENTITY soclcminstall  "<citetitle xmlns='http://docbook.org/ns/docbook'>Installing with &lcm;</citetitle>">
 <!ENTITY rhel-install  "<citetitle xmlns='http://docbook.org/ns/docbook'>Installing &rhla;</citetitle>">
 <!ENTITY soclcmplan     "<citetitle xmlns='http://docbook.org/ns/docbook'>Planning an Installation with &lcm;</citetitle>">
+<!ENTITY soclcmdeploy     "<citetitle xmlns='http://docbook.org/ns/docbook'>Deployment Guide with &lcm;</citetitle>">
```

Comment:
Only one nit: The other guide is Deployment Guide *using* < product > 

We should make it consistent.